### ISSUE TITLE
Include a link to the work in remediation LibAnswers ticket

### ISSUE THREAD

**Issue by ajkiessl** [psu-libraries/scholarsphere#1926]
Whether remediation is successful, or fails, we create a LibAnswers ticket for the accessibility team to review the documents.  Right now it just gives them the title of the Work.  We would also like a link to the work in there.
